packages/components/molecules/f-tabs/src/components/Tabs.vue

```diff
@@ -1,46 +1,142 @@
 <template>
     <div
-        :class="$style['c-tabs']"
-        data-test-id='tabs-component'>
-        {{ copy.text }}
+        data-test-id="tabs-component"
+        :class="$style['c-tabs']">
+        <div :class="$style['c-tabs-header']">
+            <div :class="$style['c-tabs-buttons']">
+                <button
+                    v-for="({ name, title }, i) in tabs"
+                    :key="i"
+                    :class="[{ [$style['c-tabs-button--active']]: activeTab === name }, $style['c-tabs-button']]"
+                    @click="selectTabIndex(name)"
+                >
+                    {{ title }}
+                </button>
+            </div>
+        </div>
+        <div :class="$style['c-tabs-body']">
+            <slot />
+        </div>
     </div>
 </template>
 
 <script>
-import { globalisationServices } from '@justeat/f-services';
-import tenantConfigs from '../tenants';
+
+const DIRECTION = {
```

Comment:
Could pull these out into a constants file – that way you could also reference it in `transitionName` in the `Tab.vue`.